### ISSUE TITLE
CanBeInteger check when value has whitespace

### ISSUE THREAD

**Issue by nguyenaiden** [bkiers/Liqp#115]
For the canBeInteger check, if the value coming in is actually a Number in string form but has leading and trailing white spaces, it is throwing a false. For example,
```
{% if foo %}
    {{foo | plus: 1}}
{% else %}
    0
{% endif %}
```
`foo` in this case actually evaluates to 1.0. 
Is adding `.trim()` to the value in canBeInteger a possible solution?

**Comment by bkiers** [comment]
Thanks for reporting @nguyenaiden!

**Comment by nguyenaiden** [comment]
Hey @bkiers! Do you know when you would be releasing the new changes?

**Comment by bkiers** [comment]
Hi @nguyenaiden 0.7.7 has just been released, so 0.7.8 will be release in a couple of weeks with (perhaps) some other fixes.

**Comment by bkiers** [comment]
@nguyenaiden FYI: https://github.com/bkiers/Liqp/issues/118
